doc-content/enterprise-only/springboot/bus-app_con.adoc

```diff
@@ -3,9 +3,7 @@
 
 Spring Framework is a Java platform that provides comprehensive infrastructure support for developing Java applications. Spring Boot is a lightweight framework based on Spring Boot starters. Spring Boot starters are `pom.xml` files that contain a set of dependency descriptors that you can include in your Spring Boot project.
 
-{PRODUCT} Spring Boot business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. Business applications are based on Spring Boot starters. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example order management or accommodation management.
-
-On the https://start.jbpm.org[business application] website you can create a Process Automation Manager, Decision Manager, or {PLANNER} business application. After you create and configure your business application, you can deploy it to an existing service or to the cloud, through OpenShift.
+{PRODUCT} Spring Boot business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. Business applications are based on Spring Boot starters. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example order management or accommodation management. After you create and configure your business application, you can deploy it to an existing service or to the cloud, through OpenShift.
```

Comment:
Please replace "On the https://start.jbpm.org[business/ application] website" with "You can create a Process Automation Manager, Decision Manager, or {PLANNER} business application using maven archetype commands " and provide the link of the maven archetype commands section, instead of removing the complete line.

Reply:
Same comment as before.

Reply:
Same comment as before.